src/Adapter/Order/CommandHandler/AddProductToOrderHandler.php

```diff
@@ -235,8 +240,13 @@ private function createOrderDetail(Order $order, ?OrderInvoice $invoice, Cart $c
      *
      * @return array
      */
-    private function getCartProductData(Cart $cart, Product $product, ?Combination $combination, int $quantity): array
-    {
+    private function getCartProductData(
+        Cart $cart,
+        Product $product,
+        ?Combination $combination,
```

Comment:
combination can be null ? you can put it at the end of the parameters

Reply:
Why at the end ?

Reply:
It's not optionnable, just nullable But the parameter is always provided

Reply:
In this case, the position should not change